browser/ui/tabs/brave_tab_strip_model.cc

```diff
@@ -0,0 +1,96 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <algorithm>
+#include <memory>
+
+#include "brave/browser/ui/tabs/brave_tab_strip_model.h"
+
+#include "brave/common/pref_names.h"
+#include "build/build_config.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/browser/web_contents.h"
+#include "ui/events/event.h"
+#include "ui/events/event_handler.h"
+
+#if defined(USE_AURA)
+#include "ui/aura/window.h"
+#endif
+
+#if defined(OS_MACOSX)
+#include "ui/views/widget/widget.h"
+#endif
+
+BraveTabStripModel::BraveTabStripModel(TabStripModelDelegate* delegate,
+                                       Profile* profile)
+    : TabStripModel(delegate, profile) {}
+
+BraveTabStripModel::~BraveTabStripModel() {}
+
+void BraveTabStripModel::SelectNextTab(UserGestureDetails detail) {
+  bool isMRUEnabled =
+      Profile::FromBrowserContext(GetActiveWebContents()->GetBrowserContext())
```

Comment:
please use `profile()` instead of `Profile::FromBrowserContext(GetActiveWebContents()->GetBrowserContext())`